### PR TITLE
[new release] fftw3 (0.8.5)

### PR DESCRIPTION
**Pull request by Chris00**
Binding to the Fast Fourier Transform library FFTW

- Project page: <a href="https://github.com/Chris00/fftw-ocaml">https://github.com/Chris00/fftw-ocaml</a>
- Documentation: <a href="https://Chris00.github.io/fftw-ocaml/doc">https://Chris00.github.io/fftw-ocaml/doc</a>

##### CHANGES:

- Update to `dune-configurator`.
- Remove the special OPAM rule for MacOS (@neynt).
- Fix deprecation warnings.

### CODE REVIEW

**Comment by Chris00** [comment]
Depends on https://github.com/ocaml/opam-repository/pull/20664

Not sure what the problem is with `dune-configurator` on opam 2.1.

File: packages/fftw3/fftw3.0.8.5/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+tags: ["FFT"]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.1"}
+  "dune-configurator"
```
Comment (mseri) [inline packages/fftw3/fftw3.0.8.5/opam:18]:
```suggestion
  "dune-configurator" {>= "1.11.4"}
```
I think it changed name there. Let's see if this fixes it, or it happened in 2.0

Comment (Chris00) [reply packages/fftw3/fftw3.0.8.5/opam]: 🎉

**Comment by mseri** [comment]
Thanks!

**Pull request merged**